### PR TITLE
Zgw to vrijbrp outgoing

### PR DESCRIPTION
**Pull request by WilcoLouwerse**
Do not merge yet, this is still a work in progress (at moment of creating this pr)
Let's check codacy

### CODE REVIEW

File: Action/vrijbrp.ZgwToVrijbrpAction.action.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "title": "ZgwToVrijbrpAction",
```
Comment (rjzondervan) [inline Action/vrijbrp.ZgwToVrijbrpAction.action.json]: Keep in mind that this is not the only VrijBRP bundle, maybe specify the name of the action a bit more

File: Mapping/zds.extraElementToEigenschap.mapping.json [hunk 2]
```diff
@@ -1,6 +1,10 @@
 {
   "title": "ZDSToZGWZaak",
+<<<<<<< HEAD:Mapping/zds.extraElementToEigenschap.mapping.json
+  "$id": "https://zds.nl/mapping/zds.extraElementToZaakEigenschap.mapping.json",
+=======
   "$id": "https://opencatalogi.nl/schemas/zds.extraElementToEigenschap.schema.json",
+>>>>>>> main:Mapping/zds.extraElementToEigenschap.schema.json
```
Comment (rjzondervan) [inline Mapping/zds.extraElementToEigenschap.mapping.json]: You've got a merge conflict here

File: Mapping/zds.extraElementToZaakEigenschap.mapping.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "ZDSToZGWZaak",
-  "$id": "https://opencatalogi.nl/schemas/zds.extraElementToZaakEigenschap.schema.json",
+  "$id": "https://zds.nl/mapping/zds.extraElementToZaakEigenschap.mapping.json",
```
Comment (rjzondervan) [inline Mapping/zds.extraElementToZaakEigenschap.mapping.json:3]: Then all references have to be changed too

File: Mapping/zds.zdsZaakToZgwZaak.mapping.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "ZDSToZGWZaak",
-  "$id": "https://opencatalogi.nl/schemas/zds.zdsZaakToZgwZaak.schema.json",
+  "$id": "https://zds.nl/mapping/zds.zdsZaakToZgwZaak.mapping.json",
```
Comment (rjzondervan) [inline Mapping/zds.zdsZaakToZgwZaak.mapping.json:3]: References

**Review by rjzondervan — CHANGES_REQUESTED** [review]
Mappings have been renamed without renaming the references to them

File: Mapping/zds.extraElementToEigenschap.mapping.json (hunk 2 above)
Comment (WilcoLouwerse) [reply Mapping/zds.extraElementToEigenschap.mapping.json]: fixed

File: Action/vrijbrp.ZgwToVrijbrpAction.action.json (hunk 1 above)
Comment (WilcoLouwerse) [reply Action/vrijbrp.ZgwToVrijbrpAction.action.json]: changed it

File: Mapping/zds.extraElementToZaakEigenschap.mapping.json (hunk 3 above)
Comment (WilcoLouwerse) [reply Mapping/zds.extraElementToZaakEigenschap.mapping.json:3]: done

File: Mapping/zds.zdsZaakToZgwZaak.mapping.json (hunk 4 above)
Comment (WilcoLouwerse) [reply Mapping/zds.zdsZaakToZgwZaak.mapping.json:3]: done

**Review by rjzondervan — APPROVED** [review]

**Pull request merged**